mfhdf/test/CMakeTests.cmake

```diff
@@ -130,44 +130,3 @@ set_tests_properties (MFHDF_TEST-hdfnctest PROPERTIES
     DEPENDS MFHDF_TEST-cdftest
     LABELS ${PROJECT_NAME}
 )
-
-#-- Adding test for xdrtest
-if (HDF4_BUILD_XDR_LIB)
-  add_executable (xdrtest ${HDF4_MFHDF_XDR_DIR}/xdrtest.c)
-  target_include_directories(xdrtest PRIVATE "${HDF4_HDFSOURCE_DIR};${HDF4_MFHDF_XDR_SOURCE_DIR};${HDF4_BINARY_DIR}")
-  if (NOT BUILD_SHARED_LIBS)
-    TARGET_C_PROPERTIES (xdrtest STATIC)
-    target_link_libraries (xdrtest PRIVATE ${HDF4_MF_XDR_LIB_TARGET})
-  else ()
-    TARGET_C_PROPERTIES (xdrtest SHARED)
-    target_link_libraries (xdrtest PRIVATE ${HDF4_MF_XDR_LIBSH_TARGET})
-  endif ()
-  if (CMAKE_C_COMPILER_ID MATCHES "IntelLLVM")
-    target_compile_options(xdrtest PRIVATE "-Wno-incompatible-function-pointer-types")
-  endif ()
-
-  if (MSVC_VERSION LESS 1900)
-    HDFTEST_COPY_FILE("${HDF4_MFHDF_XDR_DIR}/xdrtest.out" "${PROJECT_BINARY_DIR}/TEST/xdrtest.out" "xdrtest_files")
-  else ()
-    HDFTEST_COPY_FILE("${HDF4_MFHDF_XDR_DIR}/xdrtest.cyg" "${PROJECT_BINARY_DIR}/TEST/xdrtest.out" "xdrtest_files")
-  endif ()
-  add_custom_target(xdrtest_files ALL COMMENT "Copying files needed by xdrtest tests" DEPENDS ${xdrtest_files_list})
-
-  if (HDF4_ENABLE_USING_MEMCHECKER)
-    add_test (NAME MFHDF_TEST-xdrtest COMMAND ${CMAKE_CROSSCOMPILING_EMULATOR} $<TARGET_FILE:xdrtest>)
-  else ()
-    add_test (
-        NAME MFHDF_TEST-xdrtest
-        COMMAND "${CMAKE_COMMAND}"
-            -D "TEST_EMULATOR=${CMAKE_CROSSCOMPILING_EMULATOR}"
-            -D "TEST_PROGRAM=$<TARGET_FILE:xdrtest>"
-            -D "TEST_ARGS:STRING="
-            -D "TEST_FOLDER=${PROJECT_BINARY_DIR}/TEST"
-            -D "TEST_OUTPUT=xdrtest.tst"
-            -D "TEST_EXPECT=0"
-            -D "TEST_REFERENCE=xdrtest.out"
-            -P "${HDF_RESOURCES_DIR}/runTest.cmake"
-    )
-  endif ()
-  set_tests_properties (MFHDF_TEST-xdrtest PROPERTIES DEPENDS hdfnctest LABELS ${PROJECT_NAME})
-endif ()
```

Comment:
xdrtest tests the stdio flavor of XDR which isn't what the library uses and doesn't even test much. I'm ditching it so I can get rid of xdrstdio and hard-code xdrposix, in preparation for completely removing XDR.